nuclear/b.py

```diff
@@ -54,7 +54,7 @@
     # Remove lines that are comments
     if len(l) > 0 and not l.startswith("//") and not l.startswith("#"):
         # Extract our variable name from our values
-        g = re.match(r"([a-zA-Z_$][a-zA-Z_.$0-9-]*):(\w+)=(.*)", l).groups()
+        g = re.match(r"([a-zA-Z_$][a-zA-Z_ .$0-9-]*):(\w+)=(.*)", l).groups()
```

Comment:
There are really spaces in cmake variable names?
That's terrible

Reply:
yea, I think it was libzmq